assets/scss/_variables_project.scss

```diff
@@ -0,0 +1,19 @@
+/*
+
+Nothing defined here. The Hugo project that uses this theme can override theme variables by adding a file to:
+
+*/
+
+// Fonts
+
+$font-size-base: 0.65rem !default;
+
+// Dysplay styles
+$display1-size: 2.5rem !default;
+$display2-size: 2rem !default;
+$display3-size: 1.5rem !default;
+$display4-size: 1.12rem !default;
+
+// Space
+
+$spacer: 0.5rem;
```

Comment:
I'm really not fond of this, I feel it's making text way to small/difficult to read, and while I know it's easy for people to use a custom zoom with a few keybindings (like I had to do), not all users are aware and something that's easy to read should be the default.
I would personally prefer to have something less small.
Curious to hear about others / @enolfc  about this one.

Reply:
I have the exact opposite experience. It was so big on my screens, that I had to constantly use it at ~50%. To judge the scale, the Github buttons at the bottom come at a standard size, it is very hard to resize them, as the style is injected inline. Sites with those buttons look normal (meaning the site content and the size of those buttons is in harmony), whereas on our documentation site, it scrolled (out) to be usable, these buttons are almost unreadable. This to my says that the durrent documentation site is scaled wrong.

Reply:
With the new size, at 100% it's barely readable for me with my old used eyes  on my monitor (3440x1440) and laptop screen (2880x1800).
I've got to zoom to at least 135% to be able to read the text easily enough.
While for the docs.egi.eu I happily use the default zoom level (no zoom). 

I would have expected that the theme default would be something quite generic with regards to size characters to fit most setup, and if it's not we should try to find what would make the more sense/be the most versatile.
Maybe the comms team may have some suggestions?
(Too bad we can't currently have a test instance of the PR to see how they look like without having to test locally, to let people easily check...)

Reply:
So let's keep this font size and get feedback from colleagues.

Reply:
I agree with @gwarf here, this is too small to read comfortably. I am going for 125% zoom to get a good experiece
Also, lines become too long to read easily:
<img width="811" alt="Screenshot 2021-07-13 at 10 47 30" src="https://user-images.githubusercontent.com/1422438/125431306-391c983e-a236-4633-baaf-bf1cf97526af.png">

Reply:
ok, so after discussing with @thebe14, I think we can move on

Reply:
I think that we can commit like this to collect feedback from colleagues and decide if we need to increase the font.